Products/ZenModel/migrate/checkHBaseTablesExist.py

```diff
@@ -0,0 +1,50 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2016, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+import logging
+log = logging.getLogger("zen.migrate")
+
+import Migrate
+import servicemigration as sm
+sm.require("1.0.0")
+
+
+class CheckHBaseTablesExist(Migrate.Step):
+    """
+    Add existence of HBase tables to the Prereqs of OpenTSDB reader.
+    See ZEN-24094
+    """
+
+    version = Migrate.Version(5,2,0)
+
+    def cutover(self, dmd):
+        try:
+            ctx = sm.ServiceContext()
+        except sm.ServiceMigrationError:
+            log.info("Couldn't generate service context, skipping.")
+            return
+
+        # Find the services to edit.
+        # For "lite" services, there is a single opentsdb service; edit that
```

Comment:
Although it looks from your code like you aren't migrating them anyway, so just change the comment.